includes/opentsdb.inc.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/*
+ * LibreNMS
+ *
+ * Copyright (c) 2017 Yacine Ben <https://github.com/yac01/librenms.git >
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+ 
+ 
+ function opentsdb_update($device, $measurement, $tags, $fields) {
+
+        global $config, $opentsdb;      // Gloabal Variable look config.php for enable or disable OpenTSDB
+
+       if ($config['opentsdb']['enable'] == true) {   // if OpenTSDB enable
+             if ($opentsdb != true) {  // if the connection is already made
+                $opentsdb = fsockopen($config['opentsdb']['host'], $config['opentsdb']['port']);
```

Comment:
Can you add a `d_echo()` here to say that this is connecting / connected for debug purposes.